miniscope_io/models/data.py

```diff
@@ -18,7 +18,7 @@ class Frame(BaseModel, arbitrary_types_allowed=True):
     Typically returned from :meth:`.SDCard.read`
     """
 
-    data: np.ndarray
+    frame: np.ndarray
```

Comment:
This is the only breaking change - and this is only used in one place internally